e2e-tests/cypress/e2e/journeys/insurance/your-business/nature-of-business/validation/nature-of-business-goods-or-services-validation.spec.js

```diff
@@ -0,0 +1,113 @@
+
+      cy.visit(url, {
+        auth: {
+          username: Cypress.config('basicAuthKey'),
+          password: Cypress.config('basicAuthSecret'),
+        },
+      });
+
+      cy.url().should('eq', url);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  describe(`${GOODS_OR_SERVICES} error`, () => {
+    describe(`when ${GOODS_OR_SERVICES} is left empty`, () => {
+      it(`should display validation errors if ${GOODS_OR_SERVICES} left empty`, () => {
+        natureOfBusiness.goodsOrServices().clear();
+        submitButton().click();
+        partials.errorSummaryListItems().should('have.length', 1);
+        partials.errorSummaryListItems().first().invoke('text')
+          .then((text) => {
+            expect(text.trim()).equal(NATURE_OF_BUSINESS_ERRORS[GOODS_OR_SERVICES].IS_EMPTY);
+          });
+      });
+
+      it(`should focus to the ${GOODS_OR_SERVICES} section when clicking the error`, () => {
+        partials.errorSummaryListItemLinks().first().click();
+        natureOfBusiness.goodsOrServices().should('have.focus');
+      });
+
+      it(`should display the validation error for ${GOODS_OR_SERVICES}`, () => {
+        natureOfBusiness.goodsOrServicesError().invoke('text')
+          .then((text) => {
+            expect(text.trim()).equal(`Error: ${NATURE_OF_BUSINESS_ERRORS[GOODS_OR_SERVICES].IS_EMPTY}`);
+          });
+      });
+    });
+
+    describe(`when ${GOODS_OR_SERVICES} has over 1000 characters`, () => {
+      it(`should display validation errors if ${GOODS_OR_SERVICES} left empty`, () => {
+        natureOfBusiness.goodsOrServices().clear().type('a'.repeat(FIELDS.NATURE_OF_YOUR_BUSINESS[GOODS_OR_SERVICES].MAXIMUM + 1));
+        submitButton().click();
+        partials.errorSummaryListItems().should('have.length', 1);
+        partials.errorSummaryListItems().first().invoke('text')
+          .then((text) => {
+            expect(text.trim()).equal(NATURE_OF_BUSINESS_ERRORS[GOODS_OR_SERVICES].TOO_MANY_CHARACTERS);
+          });
+      });
+
+      it(`should display the validation error for ${GOODS_OR_SERVICES}`, () => {
+        natureOfBusiness.goodsOrServicesError().invoke('text')
+          .then((text) => {
+            expect(text.trim()).equal(`Error: ${NATURE_OF_BUSINESS_ERRORS[GOODS_OR_SERVICES].TOO_MANY_CHARACTERS}`);
+          });
+      });
+    });
+  });
+
+  describe(`when ${GOODS_OR_SERVICES} is correctly entered`, () => {
+    it('should not display validation errors', () => {
+      cy.visit(url, {
+        auth: {
+          username: Cypress.config('basicAuthKey'),
+          password: Cypress.config('basicAuthSecret'),
+        },
+      });
+
+      natureOfBusiness.goodsOrServices().clear().type('test');
+      submitButton().click();
+      cy.url().should('eq', turnoverUrl);
```

Comment:
Can we seperate the redireciton into a seperate `it('...)`, mentioned in another PR, same thing